lib/inspec/impact.rb

```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+
+# Impact scores based off CVSS 3.0
+module Inspec::Impact
+  IMPACT_SCORES = {
+    'none'     => 0.0,
+    'low'      => 0.01,
+    'medium'   => 0.4,
+    'high'     => 0.7,
+    'critical' => 0.9,
+  }.freeze
+
+  def self.impact_from_string(value)
+    # return if its a number
+    return value if is_number?(value)
+    raise Inspec::ImpactError, "'#{value}' is not a valid impact name [none, low, medium, high, critical]." unless IMPACT_SCORES.key?(value.downcase)
+    IMPACT_SCORES[value]
+  end
+
+  def self.is_number?(value)
```

Comment:
You could probably shorten this to:

```
value.is_a?(Integer) || value.is_a?(Float)
```

Reply:
Oh...I see...looks like you are attempting to cast as a Float then relying on that not erroring? I'm not following.

Reply:
Right, if we can cast the string as a Float then its some sort of number and we want to leave it alone.

Reply:
Take a look at the test to see how I am using this.